src/core/include/openvino/op/fake_convert.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
+namespace v13 {
+/// \ingroup ov_ops_cpp_api
+class OPENVINO_API FakeConvert : public Op {
+public:
+    OPENVINO_OP("FakeConvert", "opset13");
+
+    FakeConvert() = default;
+    FakeConvert(const ov::Output<ov::Node>& arg,
+                const ov::Output<ov::Node>& scale,
+                const ov::Output<ov::Node>& shift,
+                const std::string& destination_type = "F8E4M3",
```

Comment:
I will update it to lowercase, in case of keeping it as string, I thought that both options should be allowed (upper and lower case) and unified with `std::tolower`. We can stick with lower case only.
Currently for `ov::element` upper and lower case are acceptable:
https://github.com/openvinotoolkit/openvino/blob/d07f27205406dd1db8c69b0ab78137a0e671a610/src/core/src/type/element_type.cpp#L80-L82

Is it okay to keep the `destination_type` as string (at least for now, and update it to `ov::element` later) or it's better to introduce temporary enum representation for FP8 types attribute?

Reply:
I think it is ok.